integration/terraform/ec2/linux/main.tf

```diff
@@ -21,23 +61,37 @@ resource "aws_instance" "integration-test" {
       "cat original.pem snakeoil.pem > combine.pem",
       "sudo cp original.pem /opt/aws/amazon-cloudwatch-agent/original.pem",
       "sudo cp combine.pem /opt/aws/amazon-cloudwatch-agent/combine.pem",
+    ]
+
+    connection {
+      type        = "ssh"
+      user        = var.user
+      private_key = local.private_key_content
+      host        = aws_instance.cwagent.public_ip
+    }
+  }
+
+  #Run sanity check and integration test
+  provisioner "remote-exec" {
+    inline = [
+      "echo prepare environment",
       "export LOCAL_STACK_HOST_NAME=${var.local_stack_host_name}",
       "export AWS_REGION=${var.region}",
-      "echo run tests with the tag integration, one at a time, and verbose",
+      "export PATH=$PATH:/snap/bin:/usr/local/go/bin",
+      "echo run integration test",
       "cd ~/amazon-cloudwatch-agent",
-      "echo run sanity test && go test ./integration/test/sanity -p 1 -v --tags=integration",
+      "go test ./integration/test/sanity -p 1 -v --tags=integration",
```

Comment:
Why are we introducing this change?

Reply:
No deep reason behind it. Only format change and for cleaner code (separating stage).